### ISSUE TITLE
Does master now contain Limedrop changes?

### ISSUE THREAD

**Issue by DarkHorseDre** [kolton/d2bot-with-kolbot#1084]
Hi all, I switched to limedrop for the enhancements to trunk/master but now see master has been upgraded and Limedrop moved underneath.

Which should I be using and where can I see the status of all these changes?  tortoise took 5+ mins to return a massive list :)

**Comment by 5noop** [comment]
Limedrop has been merged with master, you can use master or unicode I guess.

**Comment by DarkHorseDre** [comment]
woot woot! thanks bro!

How can I work this out for myself (quickly)?

**Comment by DarkHorseDre** [comment]
yeah tortoise - it warned me of many changes of the branch and a request to move/delete it!
I wasn't sure if my changes would be maintained so didn't allow it, and trying to see what the changes are isn't easy (from a macro level).
Looking at the folder structure it looks as if lime was moved from branch to underneath master which is fine, but I have no idea how changes will be managed..
